.gitignore

```diff
@@ -3,3 +3,4 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+.DS_Store
```

Comment:
@OmerRahmanyGenesys  please remove .DS_Store

Reply:
@ElizaSapir  
Don't we need to add it to the gitignore?
https://pineco.de/snippets/globally-gitignore-the-ds_store-file/

Reply:
yes, we supposed to have it under `gitignore` like in all other repos we own.

Reply:
But this change is adding it to the gitignore so i don't understand the comment?
What should be done?